server/config/brain/chat/questions.fr.json

```diff
@@ -1,12 +1,75 @@
 [
   {
     "label": "chat.greeting",
-    "questions": ["Hey!"],
-    "answers": ["Hey!"]
+    "questions": ["Hey !", "Salut", "Bonjour"],
+    "answers": ["Hey !", "Salut", "Bonjour, je suis Gladys."]
+  },
+  {
+    "label": "chat.greeting-gladys",
+    "questions": ["Hey Gladys !", "Salut Gladys", "Bonjour Gladys"],
+    "answers": [
+      "Hey, je suis votre assistante domotique !",
+      "Bonjour, je suis votre assistante domotique !",
+      "Salut, je suis votre assistante domotique !"
+    ]
   },
   {
     "label": "chat.i-love-you",
     "questions": ["Je t'aime"],
     "answers": ["Moi aussi !", "Je le savais..."]
+  },
+  {
+    "label": "chat.joke",
+    "questions": [
+      "Gladys raconte moi une blague.",
+      "Gladys est ce que tu connais une blague ?",
+      "Raconte moi une blague."
+    ],
+    "answers": [
+      "Tu connais la blague du diable? Dommage elle est d’enfer !",
+      "L’autre jour, j’ai raconté une blague à mes vêtements. Ils étaient pliés !",
+      "Qui s’occupe de la décoration à l’Elysée ? Le ministère de l’intérieur.",
+      "Qu’est-ce qu’on dit quand on appelle un monstre à 4 têtes ? Allo, allo, allo, allo !",
+      "Quel est le point commun entre un facteur et un lanceur de couteau ? Les deux doivent avoir beaucoup d’adresse.",
+      "C’est l’histoire d’un chauve. Qui a un cheveu sur la langue",
+      "Pourquoi les policiers n’ont pas d’acné ? Car ils ont la peau lisse !",
+      "T’as 2 poussins, t’en veux qu’un, tu fais quoi ? T’en pousses un !"
+    ]
+  },
+  {
+    "label": "chat.shifumi",
+    "questions": [
+      "On fait un shifumi ?",
+      "Gladys, on joue au shifumi ?",
+      "shifumi",
+      "shi fu mi",
+      "Gladys, on fait un shi fu mi ?",
+      "On joue au shifumi?",
+      "On fait un pierre, papier, ciseaux ?",
+      "Gladys, on joue à pierre, papier, ciseaux ?",
+      "On joue à pierre, papier, ciseaux ?"
+    ],
+    "answers": ["Pierre !", "Papier !", "Ciseaux !"]
+  },
+  {
+    "label": "chat.heads-or-tails",
+    "questions": [
+      "On fait un pile ou face ?",
+      "Gladys, on joue à pile ou face ?",
+      "Gladys, on fait un pile ou face ?",
+      "On joue à pile ou face ?",
+      "Pile ou face"
+    ],
+    "answers": ["Pile !", "Face !"]
+  },
+  {
+    "label": "chat.numbers0to10",
+    "questions": [
+      "Donne moi un chiffre entre 0 et 10",
```

Comment:
If I ask Gladys "donne moi un chiffre entre 0 et 100", she replies with a number between 0 et 10.

Is there a way we could make this dynamic?

Reply:
I think we should use number extraction.
[(https://github.com/axa-group/nlp.js/blob/master/docs/v3/builtin-duckling.md#number-extraction)]
But I don't know how to do it.

Reply:
Two solutions: 

- Remove the feature, as without this it'll lead to weird bugs. People don't know what is your implementation, so they'll just think Gladys is broken.

- Learn how to use this library and implement the number extraction :)

Reply:
I don't think that makes Gladys crash completely.
If you prefer that I delete this part while we set up the number extraction, I understand.
I also noticed that the NLP version was obsolete, would it be interesting to upgrade to version 4, and at the same time perform number extraction?

Reply:
This is a major version , there is certainly breaking change.

Time for another PR 😉

Reply:
> If you prefer that I delete this part while we set up the number extraction, I understand. 

Yes please remove this part until the number extraction work. 

> I also noticed that the NLP version was obsolete

As Vonox said, it's not as simple as upgrading the version number, so yes: another PR.